File: backend/src/RepoFluent.Application/LifecycleAuditEntry.cs
namespace RepoFluent.Application;

public sealed record LifecycleAuditEntry(
    long Id,
    string Action,
    string ActorId,
    DateTimeOffset OccurredAt,
    string CorrelationId,
    string? PackageChecksum,
    string? PackageVersion,
    string? LifecycleStatus);

File: frontend/projects/api/src/lib/lifecycle-audit-entry.ts
export interface LifecycleAuditEntry {
  id: number;
  action: string;
  actorId: string;
  occurredAt: string;
  correlationId: string;
  packageChecksum: string | null;
  packageVersion: string | null;
  lifecycleStatus: string | null;
}
